basic login layout.html

```diff
@@ -0,0 +1,205 @@
+						<p>If you have not yet enabled your NetID or have forgotten your
+						password, or if your password may have expired, please go to <a href="http://start.nyu.edu">http://start.nyu.edu</a>.</p>
+					<button id="hideError" class="button uppercase">Back to Login</button>
+			</div>
+
+			<section class="col col2">
+				<div class="login">
+					<form action="/idp/profile/SAML2/Unsolicited/SSO?execution=e1s1" method="post" name="login" id="login">
+						<h1>NYU Login</h1>
+						            									  <p>
+						    <strong>Login to NYU Classes</strong>
+						  </p>
+												<label for="username">NetID</label>
+						<input type="text" autocomplete="off" name="j_username" id="netid" autofocus="" required="" value="">
+
+						<label for="password">Password</label>
+						<input type="password" name="j_password" id="password" required="" value="">
+						<!--
+						<input type="submit" value="Login" class="uppercase rounded" name="_eventId_proceed" >
+						-->
+						<button type="submit" style="margin:0; padding:0; display:block; width:100%; max-width:500px; height:40px; background-color:#dd1b4d; text-align:center; font:bold 18px/40px helvetica, arial, sans-serif; color:#FFF; border:0; -webkit-appearance: none; -webkit-border-radius: 3px;	-moz-border-radius: 3px; border-radius: 3px; cursor:pointer;" name="_eventId_proceed" onclick="this.childNodes[0].nodeValue='Logging in ... '">
+						Login</button>
+
+
+					</form>
+				</div>
+				<div class="help">
+					<h2>Help</h2>
+					<p>
+						<a href="https://www.nyu.edu/servicelink/041206118060462">Forgot</a>
+						your password?
+					</p>
+
+					<p>
+						<a href="https://www.nyu.edu/servicelink/KB0015994">Problems</a>
+						signing in?
+					</p>
+
+					<p>
+						Need to <a href="http://start.nyu.edu/">activate</a> your NetID or
+						NYUHome account?
+					</p>
+
+					<p>
+						For assistance: contact the <a href="http://www.nyu.edu/its/askits/helpdesk">IT Service Desk</a>
+					</p>
+				</div>
+				</section>
+				<!--/leftcol-->
+				<!--rightcol-->
+				<div class="col col2 terms">
+					<p>
+						By your use of these resources, you agree to abide by the
+						<a href="http://www.nyu.edu/about/policies-guidelines-compliance/policies-and-guidelines/responsible-use-of-nyu-computers-and-data-policy-on.html">
+						Policy on Responsible Use of NYU Computers and Data.</a>
+					</p>
+					<div><hr></div>
+                    <p>Before entering your NetID and password, verify that the URL for this page begins with: <b>https://shibboleth.nyu.edu</b></p>
+				</div>
+
+		</div>
+    </div>
+
+    <footer class="wrap">
+      <nav>
+        <ul class="clearfix">
+          <li class="left"><a href="#" class="nav-help">Need Help?</a></li>
+          <li><a href="" class="nav-login">Login</a></li>
+          <li class="right"><a href="#" class="nav-terms">Terms &amp; Privacy</a></li>
+        </ul>
+      </nav>
+    </footer>
+
+
+
+</body>
+
+</html>
```

Comment:
So as we will be using OAuth, we won't have to code this page ourselves. Users who are not logged in will be automatically taken to the NYU login page. So we don't actually need this page :)